src/frontend/packages/lib_components/src/common/ErrorComponents/BoundaryScreenError/index.tsx

```diff
@@ -129,30 +116,37 @@ export const BoundaryScreenError = ({
           pad={isSmall ? 'medium' : undefined}
           style={{ color: normalizeColor('blue-active', theme) }}
         >
-          <Onomatopoeia
-            isMobile={isSmall}
-            size={isSmall ? '2.5rem' : '4.375rem'}
-            margin="none"
+          <Text
+            weight="light"
+            className="m-0"
+            style={{
+              fontSize: isSmall ? '2.5rem' : '4.375rem',
+              letterSpacing: isSmall ? `-0.104rem;` : '-0.183rem',
+              fontWeight: '200',
+            }}
           >
-            <FormattedMessage {...messages.onomatopoeia} />
-          </Onomatopoeia>
+            {intl.formatMessage(messages.onomatopoeia)}
+          </Text>
           <Heading
             className={isSmall ? 'm-b' : 'm-xl'}
             level={2}
             fontSize={isSmall ? '3.75rem' : '9.375rem'}
           >
             {code}
           </Heading>
-          <ErrorMessage
-            maxLines={3}
-            size="0.875rem"
+          <Text
+            type="p"
+            className="m-0"
+            truncate={3}
```

Comment:
Why truncate here ?

Reply:
It replaces `maxLine` (https://v2.grommet.io/paragraph#maxLines).